readme.md

```diff
@@ -811,6 +823,21 @@ Since the GitHub API only allows 5k requests per hour, my `https://github-readme
 5.  You're done 🎉
     </details>
 
+## On docker platform
+
+<details>
+  <summary><b>:hammer_and_wrench: Step-by-step guide for deploying on docker</b></summary>
+
+  1.  Fork or clone this repo as per your needs.
+  2.  Add `express` to the dependencies section of `package.json`
```

Comment:
You can probably do this in the Dockerfile:

```docker
RUN npm install express.js
```

That way you don't add an extra dependency to the project when this dependency is necessary for the docker deployment only